frontend/packages/insights-plugin/src/components/InsightsPopup/index.tsx

```diff
@@ -19,14 +21,14 @@ const DataComponent: React.FC<DataComponentProps> = ({ x, y, datum }) => {
 };
 
 export const InsightsPopup: React.FC<PrometheusHealthPopupProps> = ({ responses, k8sResult }) => {
-  const resource = mapMetrics(responses[0].response);
-  const clusterID = (k8sResult as K8sResourceKind)?.data?.spec?.clusterID || '';
-  const riskEntries = Object.entries(resource).sort(
+  const metrics = mapMetrics(responses[0].response);
+  const clusterID = get(k8sResult as K8sResourceKind, 'data.spec.clusterID', '');
```

Comment:
Is there a reason to use lodash get over optional chaining?

Reply:
@spadgett  no special reason. Is it a worse solution than chaining? Why?

Reply:
@gkarat because using of language construction is always more preferable then using function that is doing same.

Reply:
@bond95 got it, changed :)